docs/AboutUs.md

```diff
@@ -38,15 +38,14 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 
 * Role: Developer
 
-### Jean Doe
+### Rayyan Ismail
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/rayyanismail.jpg" width="200px">
```

Comment:
Pls label the image file name with .png as required, so that the grading scripts can detect your profile image! Additionally, it shld be named with your github username (in lower case).